ClassySchedule/src/main/java/database/DatabaseStatic.java

```diff
@@ -0,0 +1,173 @@
+package database;
+
+import org.apache.hc.client5.http.classic.methods.HttpDelete;
+import org.apache.hc.client5.http.classic.methods.HttpGet;
+import org.apache.hc.client5.http.classic.methods.HttpPost;
+import org.apache.hc.client5.http.classic.methods.HttpPut;
+import org.apache.hc.client5.http.config.RequestConfig;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpResponse;
+import org.apache.hc.client5.http.impl.classic.HttpClients;
+import org.apache.hc.core5.http.HttpEntity;
+import org.apache.hc.core5.http.HttpHeaders;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.apache.hc.core5.http.io.entity.StringEntity;
+import org.apache.hc.core5.net.URIBuilder;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public final class DatabaseStatic {
+    private static final String url = "https://classy-api.ddns.net/v2/";
+    private static final JSONObject tokenObject = new JSONObject();
+
+    /**
+     * Verifies the given credentials. Populates the token object to be used
+     * for future database requests
+     * @param username the username to be checked
+     * @param password the password to be checked
+     * @return true if the user is valid, false otherwise
+     */
+    public static boolean login(String username, String password) {
+        try {
+        CloseableHttpClient client = HttpClients.createDefault();
```

Comment:
Should there be index? from within the scope?

Reply:
What do you mean?